themes/default/content/docs/get-started/kubernetes/modify-program.md

```diff
@@ -233,7 +233,10 @@ func main() {
 
         if isMinikube {
             ip = frontend.Spec.ApplyT(func(val *corev1.ServiceSpec) string {
-                return *val.ClusterIP
+                if val.ClusterIP != nil {
+				    return *val.ClusterIP
```

Comment:
Are tabs vs spaces off here?

Reply:
Yeah, I double checked my editor settings in VS Code (and they right from within the editor), so I'll take a stab at just updating them in TextEdit or something.